### DISCUSSION TITLE
Reputation custom needed

### DISCUSSION THREAD

**Discussion by Yorunina** [ejektaflex/Bountiful#254] [Mod Ideas]
Have any plan to split the reputation config into data packs? When I was creating a modpack, I found that it is a very powerful system. However, I discovered it has no entry to custom.

**Comment by ejektaflex** [comment]
I probably will not make the reputation system modular, mostly because it's a decently fragile thing that users can footgun. Changing any part of it even by a little bit can lead to an overall worse user experience (e.g. a Modpack author changes the max discount percentage from 40% to 50% in a popular Modpack, and then I get complaints about how unbalanced the mod feels at high reputation values). The current system removes this footgunning as much as possible.

However, you can still assign your own minimum reputation to objectives and rewards if you desire!
